packages/cms-lib/fileManager.js

```diff
@@ -51,6 +60,119 @@ async function uploadFolder(portalId, src, dest, { cwd }) {
   }
 }
 
+/**
+ *
+ * @param {number} portalId
+ * @param {object} file
+ * @param {string} dest
+ * @param {string} folderPath
+ */
+async function fetchFile(portalId, file, dest, folderPath) {
```

Comment:
I think we should rename this so that it is clear that it is going to fetch and write the file. Often `fetch*` is indicative of a helper for making a GET request and returning the data.